### ISSUE TITLE
Error during interval update Error: invalid

### ISSUE THREAD

**Issue by vaxi87** [sman591/homebridge-lg-thinq-ac#5]
Hi, I'm getting the following error on the Homebridge logs

`Error during interval update Error: invalid`

Let me paste the device responses:

```[5/14/2020, 7:59:10 PM] [LgThinqAirConditioner] Error during interval update Error: invalid
[5/14/2020, 7:59:10 PM] [LgThinqAirConditioner] device response {
  resultCode: '0000',
  result: {
    appType: 'NUTS',
    modelCountryCode: 'WW',
    countryCode: 'HU',
    modelName: 'RAC_056905_WW',
    deviceType: 401,
    deviceCode: 'AI01',
    alias: 'Nappali klíma',
    deviceId: '[DEVICE_ID_REMOVED]',
    fwVer: '',
    imageFileName: 'ac_home_wall_airconditioner_img.png',
    ssid: 'Kisfeszek-2.4GHz',
    softapId: '',
    softapPass: '',
    macAddress: '[MAC_ADDRESS_REMOVED]',
    networkType: '02',
    timezoneCode: 'Europe/Budapest',
    timezoneCodeAlias: 'Europe/Budapest',
    utcOffset: 1,
    utcOffsetDisplay: '+01:00',
    dstOffset: 2,
    dstOffsetDisplay: '+02:00',
    curOffset: 2,
    curOffsetDisplay: '+02:00',
    sdsGuide: '{"deviceCode":"AI01"}',
    newRegYn: 'N',
    remoteControlType: '',
    userNo: 'HU2005084425439',
    tftYn: 'N',
    deviceState: 'E',
    snapshot: {
      'airState.wMode.lowHeating': 0,
      'airState.reservation.sleepTime': 0,
      'airState.reservation.targetTimeToStart': 0,
      'airState.filterMngStates.maxTime': 0,
      'airState.opMode': 0,
      'airState.quality.sensorMon': 0,
      'airState.powerSave.basic': 0,
      meta: [Object],
      'airState.wDir.hStep': 3,
      online: true,
      timestamp: 1589479049930,
      'airState.energy.accumulated': 0,
      'airState.tempState.target': 23.5,
      'airState.miscFuncState.extraOp': 0,
      'airState.wMode.airClean': 0,
      'airState.windStrength': 8,
      static: [Object],
      'airState.tempState.current': 26,
      'airState.operation': 0,
      'airState.miscFuncState.autoDry': 0,
      'airState.filterMngStates.useTime': 0,
      'airState.wMode.jet': 0,
      'airState.energy.accumulatedTime': 0,
      'airState.reservation.targetTimeToStop': 0,
      'airState.lightingState.displayControl': 0,
      'airState.diagCode': 0,
      'airState.wDir.vStep': 0,
      'airState.energy.onCurrent': 50
    },
    online: true,
    platformType: 'thinq2',
    area: 187423,
    regDt: 20200508080455,
    blackboxYn: 'Y',
    modelProtocol: 'STANDARD',
    receipeVersion: 0,
    activeSaving: 'OFF',
    smartCareV2: 'OFF',
    order: 0,
    drServiceYn: 'N',
    regDtUtc: '20200508060455',
    groupableYn: 'Y',
    controllableYn: 'Y',
    combinedProductYn: 'N',
    masterYn: 'Y',
    tclcount: 0
  }
}
[5/14/2020, 7:59:10 PM] [LgThinqAirConditioner] device response.result.snapshot {
  'airState.wMode.lowHeating': 0,
  'airState.reservation.sleepTime': 0,
  'airState.reservation.targetTimeToStart': 0,
  'airState.filterMngStates.maxTime': 0,
  'airState.opMode': 0,
  'airState.quality.sensorMon': 0,
  'airState.powerSave.basic': 0,
  meta: { allDeviceInfoUpdate: false, messageId: '6Qjgn4O3SNOwuf7IrlB_Ng' },
  'airState.wDir.hStep': 3,
  online: true,
  timestamp: 1589479049930,
  'airState.energy.accumulated': 0,
  'airState.tempState.target': 23.5,
  'airState.miscFuncState.extraOp': 0,
  'airState.wMode.airClean': 0,
  'airState.windStrength': 8,
  static: { deviceType: '401', countryCode: 'HU' },
  'airState.tempState.current': 26,
  'airState.operation': 0,
  'airState.miscFuncState.autoDry': 0,
  'airState.filterMngStates.useTime': 0,
  'airState.wMode.jet': 0,
  'airState.energy.accumulatedTime': 0,
  'airState.reservation.targetTimeToStop': 0,
  'airState.lightingState.displayControl': 0,
  'airState.diagCode': 0,
  'airState.wDir.vStep': 0,
  'airState.energy.onCurrent': 50
}
[5/14/2020, 7:59:10 PM] [LgThinqAirConditioner] Error during interval update Error: invalid```

**Comment by sman591** [comment]
Thanks for sharing the logs- it looks like this may be happening because the snapshot state for this model is different from the LP1419IVSM model. I’ll look closer

**Comment by sman591** [comment]
@vaxi87 What air conditioner model are you using this with?

**Comment by sman591** [comment]
@vaxi87 This is happening because your AC unit responds with a fan speed of `'airState.windStrength': 8`, which I did not plan for (my unit only responds 2/4/8 for low/medium/high)

I've released 0.3.5 which fixes this so a missing fan value doesn't break the plugin -- could you try upgrading to version 0.3.5?

#8 has been opened to track adding support for additional fan speeds
